src/Microsoft.DotNet.GitHub.IssueLabeler/Models/Labeler.cs

```diff
@@ -239,9 +227,11 @@ private async Task InnerTask(string owner, string repo, int number)
             {
                 foreach (var labelFound in labels)
```

Comment:
@maryamariyan I noticed this logic doesn't end up including the area label that was found. Comments will be added for other labels to be added though. Is that a latent bug, or is that expected?